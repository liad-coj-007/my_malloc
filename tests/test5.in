add 5 5
add 6 5
eq 5 5 0 1
eq 5 6 1 1
eq 5 6 2 1
eq 6 5 3 1
print_stats
free 5
print_stats