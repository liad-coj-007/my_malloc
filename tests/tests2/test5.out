Total free bytes: 0
Total used bytes: 40
Number of free blocks: 0
Number of allocated blocks: 2
Total free bytes: 20
Total used bytes: 40
Number of free blocks: 1
Number of allocated blocks: 2